sample/Assets/Scripts/SelectAuthMethodScript.cs

```diff
@@ -12,17 +12,31 @@ public class SelectAuthMethodScript : MonoBehaviour
     [SerializeField] private Text Output;
     [SerializeField] private Button UseDeviceCodeAuthButton;
     [SerializeField] private Button UsePKCEButton;
+    string redirectUri = null;
+    string logoutRedirectUri = "https://www.immutable.com";

```

Comment:
Logout Redirect URI is only "https://www.immutable.com" for Device Code Auth flow. For PKCE it's the custom URI.